runtime/ops/tls.rs

```diff
@@ -140,8 +140,8 @@ async fn op_start_tls(
   }
 
   let tls_connector = TlsConnector::from(Arc::new(config));
-  let dnsname =
-    DNSNameRef::try_from_ascii_str(&domain).expect("Invalid DNS lookup");
+  let dnsname = DNSNameRef::try_from_ascii_str(&domain)
+    .map_err(|_| generic_error("Invalid DNS lookup"))?;
```

Comment:
Could you actually put the original rust error message into the generic_error too? Debugging the error in the JS side is very hard otherwise.

Reply:
@lucacasonato so in this case it seems there isn't a good rust error message to base it off of. 

See here for the error type we're looking at: https://briansmith.org/rustdoc/src/webpki/name.rs.html#77-82

I could update the generic error message if you have a more clear message in mind.